### ISSUE TITLE
Keyboard focus issues

### ISSUE THREAD

**Issue by rhard** [rhard/imgx#5]
@friesga There are some fixes for keyboard focus issues in the develop branch (Issue where you cannot select another X-Plane window, if some input field is active). I think it would be interesting for you to check and test it.
